dashboard/app/helpers/users_helper.rb

```diff
@@ -12,6 +12,29 @@ module UsersHelper
   ACCT_TAKEOVER_OAUTH_TOKEN = 'clever_takeover_token'
   ACCT_TAKEOVER_FORCE_TAKEOVER = 'force_clever_takeover'
 
+  # Move followed sections from old_user to new_user and destroy old_user
+  def move_sections_and_destroy_old_user(old_user, new_user)
+    # No-op if old_user is nil
+    return true unless old_user.present?
```

Comment:
Haven't thought through this, but what's the scenario if `old_user` is present but not persisted?  I remember something about `User.from_omniauth` happily returning a User instance that isn't persisted because it failed validation.

Reply:
good question. in that case, we would attempt to move any sections that `old_user` follows to `new_user`, but none should exist since the user isn't persisted. then, we would try to destroy the user, which thankfully doesn't fail for a not-persisted user.